modules/iam-profile/main.tf

```diff
@@ -53,62 +53,37 @@ resource "aws_iam_policy" "ReadOnlyPolicy" {
             "Effect": "Allow",
             "Action": [
               "apigateway:GET",
-              "codebuild:BatchGetProjects",
-	      "codebuild:ListProjects",
```

Comment:
Was this tested. We had to add this because we were getting access denied errors without it. I don't know if this was changed.

Reply:
BatchGetProjects was the main one we needed.